src/commands/verifyToken.ts

```diff
@@ -0,0 +1,26 @@
+import yargs from 'yargs';
+import { readAndParseJWK, verifyToken } from '../crypto';
+import { spinify } from '../util';
+
+export interface VerifyArguments {
+  [x: string]: unknown;
+  f: string;
+  token: string;
+}
+
+export const command = 'verify';
+
+export const describe = 'verify that a token is signed by the provided key, and return its payload';
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+export const builder: yargs.CommandBuilder<{}, VerifyArguments> = (yargs) => {
+  yargs.option('f', { alias: ['public-key-file', 'key-file'], description: 'path to load the public key (in jwk format) from', demandOption: true });
+  yargs.option('token', { alias: 't', description: 'the token to verify', demandOption: true });
+  return yargs as unknown as yargs.Argv<VerifyArguments>;
+};
+
+export const handler = async (argv: VerifyArguments): Promise<void> => {
+  const { key: publicKey, kid } = await spinify(readAndParseJWK, { message: 'reading and parsing the public key', timeout: 1500 }, argv.f);
+  const payload = await spinify(verifyToken, { message: 'verifying token', timeout: 1000 }, publicKey, argv.token, kid);
+  console.log(JSON.stringify(payload));
```

Comment:
it is better if we directly write to stdout and not use console.log

Reply:
fixed